swagger/api/swagger/swagger.json

```diff
@@ -3,7 +3,7 @@
     "info": {
```

Comment:
Seen as how we can auto-generate this now, It should be removed once we have figured out the build process end to end.